packages/plugin/src/generators/app/templates/webpack.config.js

```diff
@@ -69,10 +68,7 @@ module.exports = {
             ],
         }),
         new CopyWebpackPlugin({
```

Comment:
I don't think this is necessary anymore now that manifest.json is in ./dist.

Reply:
Just to be clear. You're saying that the CopyWebpackPlugin would just be saving maniifest.json into the dist folder where it already lives, right?

Reply:
Yeah, I assume that's what it would be doing.  In any case, the manifest is already in ./dist, which is all we really need (`pkgplugin` takes everything in ./dist and zips it up), so we shouldn't need to do anything else.